config.py

```diff
@@ -24,6 +25,7 @@ def _env(key: str, default=None) -> str:
 class UnitTestConfig:
     """For unit tests."""
 
+    AMBUDA_ENVIRONMENT = "testing"
```

Comment:
Should we also set AMBUDA_ENVIRONMENT in https://github.com/ambuda-project/ambuda/blob/main/.flaskenv and https://github.com/ambuda-project/ambuda/blob/main/scripts/install_from_scratch.sh#L36-L43? What's the difference between these two files, by the way?

Reply:
No, it's not an environment variable. It's just a hook for us to fetch the specific config version in Celery.

I find all of this `config.py` code unsatisfactory -- `_env` fails silently if the variable isn't found, e.g. -- and will revisit this in a later refactor.